pkg/queue/manager.go

```diff
@@ -571,30 +571,34 @@ func (m *Manager) getClusterQueue(cqName string) ClusterQueue {
 	return m.clusterQueues[cqName]
 }
 
-// PendingWorkloadsInfo computes a head of the new snapshot. Length of the head is set by the maxCount parameter
-func (m *Manager) PendingWorkloadsInfo(cqName string) []kueue.ClusterQueuePendingWorkload {
-	pendingWlsInfo := make([]kueue.ClusterQueuePendingWorkload, 0)
+func (m *Manager) GetPendingWorkloadsInfo(cqName string) []*workload.Info {
```

Comment:
```suggestion
func (m *Manager) PendingWorkloadsInfo(cqName string) []*workload.Info {
```

I would suggest referring to the contribution guide for SIG Scheduling.

- https://github.com/kubernetes/community/blob/master/sig-scheduling/CONTRIBUTING.md
- https://go.dev/doc/effective_go#Getters.